react/src/index.ts

```diff
@@ -10,6 +10,8 @@ import {
 
 export * from "./components";
 
+export { Embed } from "./components";
```

Comment:
IDK why I had to add this but the import didn't work without it. But there was a lot of other weird stuff happening with deps and local linking for this demo so it could be this is unnecessary now